src/fsharp/service/service.fsi

```diff
@@ -289,8 +289,9 @@ type public FSharpChecker =
     /// <param name="filename">The filename for the file.</param>
     /// <param name="options">The options for the project or script, used to determine active --define conditionals and other options relevant to parsing.</param>
     /// <param name="symbol">The symbol to find all uses in the file.</param>
+    /// <param name="canInvalidateProject">Default: true. If true, this call can invalidate the current state of project if the options have changed. If false, the current state of the project will be used.</param>
     /// <param name="userOpName">An optional string used for tracing compiler operations associated with this request.</param>
-    member FindBackgroundReferencesInFile : filename : string * options : FSharpProjectOptions * symbol: FSharpSymbol * ?userOpName: string -> Async<range seq>
+    member FindBackgroundReferencesInFile : filename : string * options : FSharpProjectOptions * symbol: FSharpSymbol * ?canInvalidateProject: bool * ?userOpName: string -> Async<range seq>
```

Comment:
If the new parameter is optional then this won’t require a major FCS version rev

Reply:
I believe it's still a binary breaking change, just not a source breaking change: there's never a member with the old signature, only a member with the new signature where callsites lacking optional members have the defaults explicitly added.

Not a showstopper IMO, just something to be aware of.